safebrowser.go

```diff
@@ -338,6 +340,8 @@ func NewSafeBrowser(conf Config) (*SafeBrowser, error) {
 		}
 	}
 
+	sb.nextlookuptime = time.Unix(0,0)
```

Comment:
How about let's skip the MinWaitDuration pieces in this change, and do it in a separate pull request.